{"name": "Test case ID: TS07", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "test_setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"87968b7b11ae4eb3385315f939608a9c\")>\n\n    @allure.description(\"validate navigation to the commercal\")\n    @allure.severity(severity_level=\"NORMAL\")\n    @allure.title(\"Test case ID: TS07\")\n    def test_6_Navigate_to_Commercial_man_with_packages(test_setup):\n        try:\n            driver = test_setup\n            hp = HomePage(driver)\n            lgn = Login(driver)\n            hp.click_save_button()\n            lgn.move_to_login()\n            lgn.login(\"0586691039\")\n            lgn.submit_login()\n            lgn.login_input_squares()\n            lgn.submit_secret_code()\n            hp.commercial_man_with_packages()\n            assert hp.assert_commercial_man_with_packages()\n        except Exception as e:\n            print(e)\n            driver.save_screenshot(\"test_TS07_error.png\")\n>           raise e\n\ntests\\test_homepage.py:114: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntests\\test_homepage.py:109: in test_6_Navigate_to_Commercial_man_with_packages\n    hp.commercial_man_with_packages()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Src.pages.HomePage.HomePage object at 0x000001699F89C410>\n\n    def commercial_man_with_packages(self):\n        commercial_xpath = '//*[@id=\"root\"]/div/div[2]/div[2]/div/div/div[2]/div/div[2]/div[1]/div/div/div/div/ul/li[5]/a'\n        element = self.driver.find_element(By.XPATH, commercial_xpath)\n        self.driver.execute_script(\"arguments[0].click();\", element)\n>       self.driver.switch_to.window(self.driver.window_handles[1])\nE       IndexError: list index out of range\n\nSrc\\pages\\HomePage.py:91: IndexError"}, "description": "validate navigation to the commercal", "attachments": [{"name": "stdout", "source": "e239b17e-3387-41d6-8965-813ab975ff49-attachment.txt", "type": "text/plain"}], "start": 1680424761932, "stop": 1680424763750, "uuid": "5086ed8f-f1dd-46a5-81a9-09db988e317c", "historyId": "ad090bb1e9e6bf117fb2f9ee24b921f3", "testCaseId": "24de231b8be29eef60103336b8048a0b", "fullName": "tests.test_homepage#test_6_Navigate_to_Commercial_man_with_packages", "labels": [{"name": "severity", "value": "NORMAL"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_homepage"}, {"name": "host", "value": "DESKTOP-9HU6DH7"}, {"name": "thread", "value": "34884-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_homepage"}]}